### PR TITLE
Add Placement Policy and Custom Properties support to vRA Project functions

### LINKED ISSUE #267: Feature Request - Projects Custom Properties
Would be **fantastic** to have a way to work with the custom properties of projects.

### PR DESCRIPTION
**Pull request by jonathanmedd**
`New-vRAProject` and `Get-vRAProject`:

- Added Placement Policy and Custom Properties support

`Update-vRAProject`

- Renamed to `Set-vRAProject`
- Added alias of `Update-vRAProject` for backwards compatibility
- Re-written to support examining existing vRA project and only changing properties specified by user
- Added support for Placement Policy and Custom Properties
- Added support to supply a new Project name to rename to

To close #267

### CODE REVIEW

File: src/Functions/Public/iaas/New-vRAProject.ps1 [hunk 1]
```diff
@@ -30,6 +30,12 @@
     .PARAMETER SharedResources
     Deployments are shared between all users in the project
 
+    .PARAMETER PlacementPolicy
+    Placement Policy
```
Comment (Stevio54) [inline src/Functions/Public/iaas/New-vRAProject.ps1:34]: Just a suggestion to add (e.g. DEFAULT or SPREAD) as I see it in the validation set but maybe for documentation purposes note it here?

File: src/Functions/Public/iaas/Set-vRAProject.ps1 [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+                if (!$ExistingProject){
+
+                    throw "Project with name $($Name) does not exist"
+                }
+                $Id = $ExistingProject.Id
+            }
+
+            # --- Update Project by JSON
+            'JSON' {
+                $Data = ($JSON | ConvertFrom-Json)
```
Comment (Stevio54) [inline src/Functions/Public/iaas/Set-vRAProject.ps1:244]: Parenthesis not needed here

**Review by Stevio54 — CHANGES_REQUESTED** [review]
Just a few small nit-picky things but looks good otherwise. I tested this with a few scenarios in a test environment I have and seems to work very well :) Great job on this.

File: src/Functions/Public/iaas/New-vRAProject.ps1 (hunk 1 above)
Comment (jonathanmedd) [reply src/Functions/Public/iaas/New-vRAProject.ps1]: Have updated with extra text

File: src/Functions/Public/iaas/Set-vRAProject.ps1 (hunk 2 above)
Comment (jonathanmedd) [reply src/Functions/Public/iaas/Set-vRAProject.ps1]: Have removed parentheses

**Comment by jonathanmedd** [comment]
@Stevio54 Thanks for the feedback, have implemented changes.

FYI seems like the Pester version has updated and re-implemented the Code Coverage feature which broke the build since we do not have the configuration for it in the pipeline. Have temporarily disabled it in this PR and will open an issue to implement it correctly.

**Review by Stevio54 — APPROVED** [review]
This looks great!

**Pull request merged**